sql/core/src/test/scala/org/apache/spark/sql/execution/metric/SQLMetricsSuite.scala

```diff
@@ -82,11 +82,14 @@ class SQLMetricsSuite extends SharedSparkSession with SQLMetricsTestUtils {
   test("Filter metrics") {
     // Assume the execution plan is
     // PhysicalRDD(nodeId = 1) -> Filter(nodeId = 0)
-    val df = person.filter('age < 25)
-    testSparkPlanMetrics(df, 1, Map(
-      0L -> (("Filter", Map(
-        "number of output rows" -> 1L))))
-    )
+    Seq((0L, false), (1L, true)).foreach { args =>
```

Comment:
For better readability, could you use named variables, e.g., `.foreach { case (xxx, yyy) =>`?

Reply:
fixed in the new commit